### DISCUSSION TITLE
Editor de código

### DISCUSSION THREAD

**Discussion by UncannyChimp** [Launch-X-Latam/MisionFrontEnd#1] [Q&A] [answered]
Para todo lo que se presentara en este curso, podemos usar visual studio code como editor de còdigo??

**Comment by romarpla** [comment, marked as answer]
Si, puedes usar VS code, Atom, Sublime o con el que te sientas más cómodo

**Reply by Kris18274** [reply]
Incluso, Notepad también.

**Reply by AlvaroFierro** [reply]
Para los mas rudos, usen vim 😆

**Reply by FluffyRaccon** [reply]
> Incluso, Notepad también.

Yo uso Notepad++ ya que no soy tan rudo jeje.

**Reply by atzsamuel** [reply]
windows notepad xd

**Reply by Rubensmorph** [reply]
Usen Notepad++ 👍

**Reply by EduGarcia502** [reply]
yo uso el block de notas, es mejor

**Reply by OscarTinajero117** [reply]
100% recommended: 
https://www.youtube.com/watch?v=X34ZmkeZDos
(It's a joke: v)

**Reply by pierowebdesign** [reply]
Sublime Text, Atom, Notepad++ son buenas opciones, personalmente me gusta el Sublime Text

**Reply by LaelInnovaccion** [reply]
VS code

**Comment by Murillas** [comment]
Gracias Lord Commander

**Comment by jamescovenant2020** [comment]
podemos tomar notas sobre los temas poco a poco?

**Reply by OscarNoe** [reply]
Te recomiendo hacer notas a mano

**Reply by jamescovenant2020** [reply]
vale gracias

**Reply by hnolascos1600** [reply]
Te recomiendo hacer notas en Onenote.

**Reply by UncannyChimp** [reply]
Primero, conocedor de halo verdad?, y segundo, mas que notas cuando puedas práctica todo, sirve más que las notas en mi caso

**Reply by AlvaroFierro** [reply]
Pueden hacer sus notas en un repo y hacer readmes, por si luego nos quitan de la organizacion

**Reply by FluffyRaccon** [reply]
Si tienes un smartphone, y te sientes cómodo con los servicios de Google, te sugiero Google Keep, te mantiene tus dispositivos sincronizados con tus notas.

**Reply by OscarNoe** [reply]
Notion tambien es opcion para hacer notas

**Reply by JesusJimenez4** [reply]
Si, y si en la sesión no alcanzas a hacer las notas bien,  puedes volver a ver la clase, ya que la sesión se queda grabada.

**Reply by ALHnosre** [reply]
Para que te vayas acostumbrando, ve tomando notas directamente en el editor de texto que vayas a usar, no se, si usas VSCode, crea un archivo txt y alli vas tomando tus notitas, o no se, un archivo html, y vas dejando tus notitas alli, para familiarizarte con las cosas de la mejor manera

**Comment by JosLuisDev** [comment]
Si, de hecho, viene en los materiales del repositorio los IDEs o editores de código que puedes utilizar.

**Comment by ghost** [comment]
También pueden usar un editor de código basado en la nube como CODESANDBOX si su computadora no tiene mucha memoria o es algo lenta. Búsquenlo en internet.

**Comment by AlvaroFierro** [comment]
Tambien pueden usar VS Code en la nube, desde [vscode.dev](https://vscode.dev/)

**Comment by magouwuxd** [comment]
puedes hacerlo hasta en papel

**Reply by Rubensmorph** [reply]
Ahora falta el compilador

**Reply by AlvaroFierro** [reply]
JavaScript no necesita compilador, ya que es un lenguaje de script ademas de que es de alto nivel

**Comment by OATS74** [comment]
Buenas noches por favor facilitarme el link de la clase de hoy . Gracias y ms team  para que es no veo a nadie conectado alli

**Comment by rgplazas** [comment]
Uso Visual Studio Code

**Comment by erykzon** [comment]
Puedes usar VS code pero si no quieres instalar nada les recomiendo [REPLIT] (https://replit.com/@erykzon)

**Comment by dllg-lx** [comment]
Yo varío entre dos:
Si quiero un entorno rápido y cómodo utilizo VS code
Si me quiero divertir explorando, personalizando y aprendiendo cosas nuevas entonces utilizo NeoVim
